pkgs/development/python-modules/decorating/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy27, isPy34, isPy35, isPy36 }:
+
+assert (isPy27 || isPy34 || isPy35 || isPy36);
```

Comment:
this should be removed and replaced with a `disabled = ...;` argument passed to `buildPython*`